source/agora/node/Ledger.d

```diff
@@ -526,6 +526,7 @@ public class Ledger
     protected void updateValidatorSet (in Block block) @safe
     {
         this.enroll_man.clearExpiredValidators(block.header.height);
+        this.enroll_man.updateValidatorIndexMaps(Height(block.header.height));
```

Comment:
This should only be called if the height is greater than 0. Otherwise we get errors logged for no validators at height 0.

Reply:
apart from that I think we need to handle the case where 
no utxo_set is provided and 'this.enroll_man.validatorCount() != 0'

Reply:
I believe it is handled within this function [this.addValidatedBlock(block);](https://github.com/bpfkorea/agora/blob/5310eb4bec8dac40a00ac7fe8568eee45495173e/source/agora/node/Ledger.d#L191) which is called for each block in the case you mention.